crates/bindings-csharp/Runtime/Runtime.csproj

```diff
@@ -26,7 +26,7 @@
 
   <ItemGroup>
     <None Include="README.md" Pack="true" PackagePath="" />
-    <None Include="build/" Pack="true" PackagePath="build" />
+    <None Include="build/*" Pack="true" PackagePath="build" />
```

Comment:
Is this a recursive glob?

Reply:
No, but we don't have any nesting there, just 2 files.